crates/evm/src/evm/executor.rs

```diff
@@ -80,6 +81,11 @@ pub(crate) fn execute_multiple_tx<
             Err(EVMError::Transaction(
                 InvalidTransaction::CallerGasLimitMoreThanBlock,
             ))
+        } else if tx.signer() == SYSTEM_SIGNER {
+            Err(EVMError::Custom(format!(
+                "Ignored system transaction: {:?}",
```

Comment:
call this invalid system transaction

Reply:
Why not retain !sys txs from `users_txs` before even passing then into executor::execute_multiple_tx?

Reply:
> Why not retain !sys txs from users_txs before even passing then into executor::execute_multiple_tx?

I didn't understand this. 

But this is a fix for the following case:

Sequencer creates a system transaction and puts it along with other mempool transactions.

We execute them for free.

So where we process user txs, there shouldn't be any system txs.

Reply:
Is it clear now @kpp ?

Reply:
Discussed in slack